helm/fiftyone-teams-app/README.md

```diff
@@ -18,104 +18,79 @@
+
 <!-- markdownlint-enable line-length -->
 
+This page assumes general knowledge of FiftyoneTeams and how to use it.
 Please contact Voxel51 for more information regarding Fiftyone Teams.
 
-## FiftyOne Teams v2.0+ Requires a License File
-
-FiftyOne Teams v2.0 introduces a new requirement for a license file.  This
-license file should be obtained from your Customer Success Team before
-upgrading to FiftyOne Teams 2.0 or beyond.
-
-The license file now contains all of the Auth0 configuration that was
-previously provided through kubernetes secrets; you may remove those secrets
-from your `values.yaml` and from any secrets created outside of the Voxel51
-install process.
-
-Use the license file provided by the Voxel51 Customer Success Team to create
-a new license file secret:
+## Table of Contents
 
-```shell
-kubectl --namespace your-namepace-here create secret generic fiftyone-license \
---from-file=license=./your-license-file
-```
+<!-- toc -->
 
-## FiftyOne Teams v2.2+ Delegated Operator Changes
+- [Requirements](#requirements)
+  - [Kubernetes/Kubectl](#kuberneteskubectl)
+  - [Helm](#helm)
+- [Usage](#usage)
+- [Initial Installation vs. Upgrades](#initial-installation-vs-upgrades)
+- [Advanced Configuration](#advanced-configuration)
+  - [Builtin Delegated Operator Orchestrator](#builtin-delegated-operator-orchestrator)
+  - [Central Authentication Service](#central-authentication-service)
+  - [FiftyOne Teams Authenticated API](#fiftyone-teams-authenticated-api)
+  - [Plugins](#plugins)
+  - [Proxies](#proxies)
+  - [Snapshot Archival](#snapshot-archival)
+  - [Storage Credentials and `FIFTYONE_ENCRYPTION_KEY`](#storage-credentials-and-fiftyone_encryption_key)
+  - [Text Similarity](#text-similarity)
+- [Values](#values)
+  - [Deploying On GKE](#deploying-on-gke)
 
-FiftyOne Teams v2.2 introduces some changes to delegated operators, detailed
-below.
+<!-- tocstop -->
 
-### Delegated Operation Capacity
+## Requirements
 
-By default, all deployments are provisioned with capacity to support up to 3
-delegated operations simultaneously. You will need to configure the [builtin
-orchestrator](#builtin-delegated-operator-orchestrator) or an external
-orchestrator, with enough workers, to be able to utilize this full capacity.
-If your team finds the usage is greater than this, please reach out to your
-Voxel51 support team for guidance and to increase this limit!
+Helm and Kubectl must be installed and configured on your machine.
 
-### Existing Orchestrators
+### Kubernetes/Kubectl
 
-> [!NOTE]
-> If you are currently utilizing an external orchestrator for delegated
-> operations, such as Airflow or Flyte, you may have an outdated execution
-> definition that could negatively affect the experience. Please reach out to
-> Voxel51 support team for guidance on updating this code.
+The following kubernetes/kubectl versions are required:
 
-Additionally,
+Kubernetes: `>=1.18-0`
```

Comment:
Why is this two lines?  Why not just: 'Kubernetes versions greater than, or equal to, 1.18-0 are supported using the Voxel51 Helm Chart.' or somesuch?

Reply:
We use the helm templates to fill that out:

```shell
{{ template "chart.kubeVersionLine" . }}
```